### PR TITLE
Update travis config

### PR DESCRIPTION
**Pull request by indrekj**
Upgrades ruby and adds jruby.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 sudo: false
 language: ruby
 rvm:
-- 2.3.1
+  - 2.4.1
```
Comment (bhs) [inline .travis.yml]: I'm not an experienced Ruby programmer – does bumping to 2.4.x make it difficult for us to verify compatibility with 2.3.x?

**Comment by indrekj** [comment]
@bhs Updated, added both: ruby 2.3 and 2.4. Results: https://travis-ci.org/indrekj/opentracing-ruby/builds/217617494 . Just noticed that jruby build is failing, will take a look at it tomorrow.

**Comment by indrekj** [comment]
All green now https://travis-ci.org/indrekj/opentracing-ruby/builds/217747465

**Comment by indrekj** [comment]
@bhs could you merge please. Also please go to Travis settings https://travis-ci.org/profile/opentracing and enable it for `opentracing-ruby` repository.

**Comment by bhs** [comment]
thank you @indrekj, and I'm glad you pinged me, I am swamped. :-/

**Pull request merged**